core/primitives/src/account.rs

```diff
@@ -128,6 +145,16 @@ struct LegacyAccount {
     storage_usage: StorageUsage,
 }
 
+#[cfg(feature = "protocol_feature_add_account_versions")]
+#[derive(BorshSerialize, BorshDeserialize)]
+struct SerializableAccount {
```

Comment:
should we implement `From<SerializableAccount>` for `Account`?

Reply:
Considering it should ever be used only once I am not sure we need this